### ISSUE TITLE
As a User/PA/Liason/Admin I want a record of changes to a request's Status to appear at the bottom of the Details modal on the Bookings table so that I can see who took which action, when (and presumably why).

### ISSUE THREAD

**Issue by nopivnick** [ITPNYU/booking-app#450]
![Image](https://github.com/user-attachments/assets/cce2db9e-81f3-4549-84c6-9b83eecf48cc)

**Comment by nopivnick** [comment]
@rlho you should connect with @Greenfishyy if you are implementing the UI piece of this, I believe she's working on a booking details modal redesign.

**Comment by lucia-gomez** [comment]
can i mark this as review or are we waiting on final UI design?

**Comment by nopivnick** [comment]
@lucia-gomez mark it as status = Review. we'll treat implementing @Greenfishyy 's design as a separate task.
